addons/mail/static/src/core/common/thread.js

```diff
@@ -556,6 +556,9 @@ export class Thread extends Component {
         if (msg.parentMessage) {
             return false;
         }
+        if (this.env.inChatWindow && msg.is_note) {
+            return false;
+        }
```

Comment:
`msg.is_note` should be enough: we don't want to squash them in all contexts. In chatter this is already handled by `this.env.inChatter` above.